docs/examples/101/create-managedidentity-rbac/main.bicep

```diff
@@ -0,0 +1,19 @@
+param managedIdentityName string
+param location string
+param roleDefinitionId string = 'b24988ac-6180-42a0-ab88-20f7382dd24c' //Default as contributor role
+param roleAssignmentName string
+
+resource msi 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+  name: managedIdentityName
+  location: location
+}
+
+resource roleassignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
+  name: guid(roleAssignmentName)

```

Comment:
I'd recommend making this guid more unique with additional arguments. Usually, we suggest the triplicate of role definition id, prinicpal id, and scope, so that would leave you with:

```
name: guid(roleDefinitionId, msi.properties.principalId, resourceGroup().id)
```